<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>휴머노이드 기본 제어</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2" colspan="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_1">
        <property name="text">
         <string>인사</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>승리</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>하이</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>좌절</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>파이트</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>포트찾기</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPort">
        <property name="text">
         <string>None</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="1" colspan="1">
     <layout class="QVBoxLayout" name="rightLayout">
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>행동하기</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_name">
        <property name="text">
         <string>이름</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ed_name"/>
      </item>
      <item>
       <widget class="QLabel" name="lb_phone">
        <property name="text">
         <string>전번</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ed_phone"/>
      </item>
      <item>
       <widget class="QLabel" name="lb_memo">
        <property name="text">
         <string>메모</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="ed_memo"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="saveDeleteLayout">
        <item>
         <widget class="QPushButton" name="btn_save">
          <property name="text">
           <string>저장</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_del">
          <property name="text">
           <string>삭제</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="camExitLayout">
        <item>
         <widget class="QPushButton" name="btn_cam">
          <property name="text">
           <string>촬영</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_exit">
          <property name="text">
           <string>종료</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2" rowspan="3" colspan="1">
     <widget class="QLabel" name="lb_video">
      <property name="text">
       <string>video</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>포트찾기</string>
    </property>
    <addaction name="actionSerial_Port"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSerial_Port">
   <property name="text">
    <string>Serial Port</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
